Sources/App/Views/PublicPage.swift

```diff
@@ -120,8 +120,7 @@ class PublicPage {
     final func description(maxLength: Int = 200) -> String {
         guard let pageDescription = pageDescription() else {
             return """
-            The Swift Package Index is the place to find the best Swift packages. \
-            Indexing 2,482 packages and 36,483 versions.
+            The Swift Package Index is the place to find the best Swift packages.
```

Comment:
I've had to move this to `HomeIndex`, because we don't have a model available in the generic class. We could make the `request` available and thereby get a `db` with which to query but I think we only really care about the stats for links to the home page anyway.

We can add the stats to other pages that have models quite easily as well but I've left it at that.

Reply:
Yea that works. Thanks